SKYKitChat/Classes/UI/ConversationView/SKYChatConversationViewController.swift

```diff
@@ -0,0 +1,160 @@
+//
+//  SKYChatConversationViewController.swift
+//  SKYKitChat
+//
+//  Copyright 2016 Oursky Ltd.
```

Comment:
I don't know whether we should update all the copyright declaration. Or Just update it when a new file is created?

Any convention?